src/drools/ruleset.py

```diff
@@ -91,6 +91,9 @@ def _make_jpy_instance():
 
     jvm_options.extend(jvm_log_options)
 
+    log_delay = os.environ.get("DROOLS_LOG_DELAY", "5")
```

Comment:
@tkobayas do we need this at all. On your side you couldn't we just use the Java System.getEnv() or is jvm_options the only way to pass it in. Since this is purely for diagnostic purpose on the container launch we can set an env var and it should be available to you for logging